newIDE/app/src/stories/index.js

```diff
@@ -3876,12 +3876,15 @@ storiesOf('ProfileDetails', module)
   .addDecorator(muiDecorator)
   .add('profile', () => (
     <ProfileDetails
-      profile={indieUserProfile}
+      authenticatedUser={fakeIndieAuthenticatedUser}
       onEditProfile={action('edit profile')}
     />
   ))
   .add('loading', () => (
-    <ProfileDetails profile={null} onEditProfile={action('edit profile')} />
+    <ProfileDetails
+      authenticatedUser={fakeAuthenticatedButLoadingAuthenticatedUser}
+      onEditProfile={action('edit profile')}
+    />
   ));
 
```

Comment:
Can you add a story for `EmailVerificationPendingDialog` with:

- fakeAuthenticatedButLoadingAuthenticatedUser (i.e: "loading" state)
- fakeIndieAuthenticatedUser (i.e: "not email verified user")
- fakeIndieAuthenticatedAndEmailVerifiedUser (a new fixture to add :)) (i.e: "email verified user").

This story will help to quickly ensure the design of the dialog is ok when we refactor/change things later (because this is the kind of dialog we don't see every day, so it's especially important they have a story so that we can visit them easily to ensure they are doing good :))

Reply:
added 2 stories, turns out the "non-verified user" is the same as the one "loading" :) (and waiting to be verified!)